### DISCUSSION TITLE
[help] How to get config.yaml changes into existing installations

### DISCUSSION THREAD

**Discussion by djdookie** [Chia-Network/chia-blockchain#3482] [Support]
Version 1.1.3 lately brought some additions and a lot of cleanup and comments to config.yaml.
I'd like to have this new config.yaml in all my installations.
I included the changes by hand now, but is there a way to get an updated config.yaml without having to do a fresh install and losing previos config changes?
Do I have to delete config.yaml and make a "chia init" again, then make changes to the configuration again?

I recognized changes to config.yaml like changes to number of in- and outgoing peers don't come to existing clients by itself. Or max. number for debug rotation files.

**Comment by heus12** [comment]
bakcup config.yaml first!

**Reply by djdookie** [reply]
Yeah sure. So I have to merge configs manually?

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
